app/src/main/kotlin/com/wire/android/ui/authentication/login/sso/LoginSSOViewModel.kt

```diff
@@ -44,6 +60,47 @@ class LoginSSOViewModel @Inject constructor(
         }
     }
 
+    fun establishSSOSession(ssoLoginResult: DeepLinkResult.SSOLogin.Success) {
```

Comment:
please add the unit test for this method

Reply:
W can only pass the cookie here if we only need it, why DeepLinkResult.SSOLogin.Success ?